src/pages/dtrader/DtraderTabs.tsx

```diff
@@ -7,6 +7,10 @@ import device from 'themes/device.js'
 import { isBrowser } from 'common/utility'
 import { LinkButton } from 'components/form'
 
+type StepCommonType = {
```

Comment:
You have to name files component files inside the `pages` directory with a leading `_` otherwise it will create a page for it. In this case, you have a page create at `/dtrader/DtraderTabs`

Reply:
it was pointed out by SonarCloud as code smell, let revert it back.

Reply:
@alamin-deriv SonarCloud flagged it out because the component name is DtraderTabs. So instead, you could either rename the file to `_dtrader-tabs.js` or rename the component to `DtraderVideo`.